go.sum

```diff
@@ -365,6 +365,8 @@ github.com/onsi/gomega v1.19.0 h1:4ieX6qQjPP/BfC3mpsAtIGGlxTWPeA3Inl/7DtXw1tw=
 github.com/onsi/gomega v1.19.0/go.mod h1:LY+I3pBVzYsTBU1AnDwOSxaYi9WoWiqgwooUqq9yPro=
 github.com/openshift-online/ocm-sdk-go v0.1.288 h1:lhA5YLgGP2m23xiMKIivD/T/J3zt7ee0UWbzDxm6HmU=
```

Comment:
this line and the one below should not be needed anymore I think, a go mod tidy should clean it up